src/role-access/containers/exclusion/describe-exclusion/describe-exclusion.component.spec.ts

```diff
@@ -1,15 +1,14 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
 import { provideMockStore } from '@ngrx/store/testing';
-import { DescribeExclusionComponent } from './describe-exclusion.component';
+import { DescribeExclusionContainerComponent } from './describe-exclusion.component';
 
 describe('DescribeExclusionComponent', () => {
-  let component: DescribeExclusionComponent;
-  let fixture: ComponentFixture<DescribeExclusionComponent>;
+  let component: DescribeExclusionContainerComponent;
```

Comment:
Assume this will change once 'Container' has been removed

Reply:
Yes this is already moved